app/src/main/java/com/dsvag/currencyexchanger/data/utils/KeyBoardUtils.kt

```diff
@@ -0,0 +1,30 @@
+package com.dsvag.currencyexchanger.data.utils
+
+import android.content.Context
+import android.util.Log
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import androidx.core.content.ContextCompat
+
+class KeyBoardUtils(context: Context) : ContextCompat() {
+    private val TAG = KeyBoardUtils::class.simpleName
+
+    private val inputMethodManager
+            by lazy { getSystemService(context, InputMethodManager::class.java)!! }
```

Comment:
У тебя опять зависимость создается внутри - вынеси в конструктор
```suggestion
class KeyBoardUtils(
    private val inputMethodManager: InputMethodManager,
) {
    private val TAG = KeyBoardUtils::class.simpleName
```